Autopilot/TelemetryManager/Inc/telemetryStateClasses.hpp

```diff
@@ -19,6 +24,7 @@ class initialMode: public telemetryState
         initialMode(){}
         initialMode(const initialMode& other);
         initialMode& operator =(const initialMode& other);
+        void TelemCommWithPMInit();
```

Comment:
You do not need to add `TelemCommWithPMInit()` here. In fact by doing this you will now call this undefined function (which is incorrect) whenever you write `TelemCommWithPMInit()`. 

Since you are importing `CommsWithPathManager.hpp` you will have access to this function and will just have to call `TelemCommWithPMInit()` to call the right one. 

Tl;dr: delete this

Reply:
in d124597